src/components/DatePicker/readme.md

```diff
@@ -240,5 +240,76 @@ const containerStyles = {
         value={state.date}
         onChange={date => setState({ date })}
     />
-</div>
+</div>;
+```
+
+##### DatePicker select date:
+
+```js
+import React from 'react';
+import { Picklist, PicklistOption, DatePicker, Input, Avatar } from 'react-rainbow-components';
+import styled from 'styled-components';
+//import img from './icons/avatar-5.svg';
+
+const initialState = {
+    date: '',
+    locale: { name: 'en-US', label: 'English (US)' },
+};
+
+const Title = styled.h2`
+    color: ${props => props.theme.rainbow.palette.text.title};
+    font-size: 2rem;
+    margin: 0 0.5rem 0 1rem;
+`;
+
+const Name = styled.h1`
+    color: ${props => props.theme.rainbow.palette.text.title};
+    font-size: 2rem;
+    margin: 0 0.5rem 0 1rem;
+    font-weight: 800;
+`;
+
+const Email = styled.p`
+    color: ${props => props.theme.rainbow.palette.text.title};
+    font-size: 1rem;
+    margin: 0 0.5rem 0 1rem;
+`;
+
+const StyledAvatar = styled(Avatar)`
+    width: 100px;
+    height: 100px;
+    box-shadow: 0 2px 4px 0 rgba(6, 28, 63, 0.3);
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+`;
+
+<div className="rainbow-align-content_center rainbow-m-vertical_large rainbow-p-horizontal_small rainbow-m_auto rainbow-flex_column">
+    <div className="rainbow-m-vertical_medium">
+        <Title>Complete your Information</Title>
+    </div>
+    <div className="rainbow-flex rainbow-align_center rainbow-m-vertical_medium">
+        <div>
+            <StyledAvatar src="images/user/avatar-5.svg" assistiveText={name} title={name} />
+        </div>
+        <div>
+            <Name>Ana Doe</Name>
+            <Email>ana@gmail.com</Email>
+        </div>
+    </div>
+    <div className="rainbow-m-vertical_medium" style={{ width: '400px' }}>
+        <DatePicker
+            id="datePicker-1"
+            placeholder="Select your DOB"
+            value={state.date}
+            onChange={value => setState({ date: value })}
+            label="Birthday"
+            formatStyle="medium"
+            locale={state.locale.name}
```

Comment:
since this locale is fixed and it will not change then we can put it here directly instead of using a state:
```locale="'en-US"```
then remove it from initialState